src-old/snake.hpp

```diff
@@ -7,7 +7,7 @@ namespace snake
     private:
         /* game variables */
         // stores the snakes position
-        unsigned short int snakePos[2][100]{0};
+        unsigned short int snakePos[2][100]({0});
```

Comment:
why ist there a change in the old source files, that wouldn't work anyway? undo that, even if we'll probably delete that file in some time.